.vscode/settings.json

```diff
@@ -1,33 +1,31 @@
 {
-    "python.formatting.provider": "black",
```

Comment:
Are the missing fields here no longer needed?

Reply:
@mvadari Yes, they're deprecated